packages/web-components/src/player-component/player.class.ts

```diff
@@ -683,16 +682,16 @@ export class PlayerWrapper {
                 const videoCurrentTime = video.currentTime; // Lock in the current value
                 const videoBufferedEnd = videoBuffered.length > 0 ? videoBuffered[videoBuffered.length - 1].end : videoCurrentTime - 1;
                 if (videoBufferedEnd - MAX_LATENCY_WINDOW > videoCurrentTime) {
-                // TODO: To account for the time that elapses during the trickplay portion, we need to add some additional trickplay duration
-                // in order to get latency as low as possible. The slower the trickplay speed, and the higher the latency, the higher this
-                // additional value will have to be. For now we use a flat value.
-                    const additionalDur= 2000; // in milliseconds
+                    // TODO: To account for the time that elapses during the trickplay portion, we need to add some additional trickplay duration
```

Comment:
I don't have a problem with this whitespace-only change but I am wondering how it got through. I guess lint doesn't check for indent? Otherwise the offending PR would have failed validation. In an ideal world this would be a separate commit and the PR would be completed using rebase and fast forward but time crunch being what it is...

Reply:
we use prettier in our team, so on save the file auto formats with correct formatting. I can checkin the rules and recommended vsCode extensions in a later PR, so that everyone is using the same tools and rules.